.eslintrc.json

```diff
@@ -1,20 +0,0 @@
-{
-    "env": {
-        "browser": true,
-        "commonjs": true,
-        "es2021": true
-    },
-    "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
-    "parser": "@typescript-eslint/parser",
-    "parserOptions": {
-        "ecmaVersion": "latest",
-        "project": "./tsconfig.json"
-    },
-    "plugins": ["@typescript-eslint"],
-    "rules": {
-        "@typescript-eslint/no-unused-vars": "off",
```

Comment:
@pendulum-chain/devs any idea why we had disable this unused variables rule? I enabled it in the new config. But I could be missing something.